model-terms/model-terms-v1.1.md

```diff
@@ -0,0 +1,398 @@
+
+8. Each Contracting Party shall ensure that:
+
+	i. any Party that uses the User Interface includes a link or other visible mechanism that permits access and use of the User Interface on every webpage accessible to Users; and
+
+	ii. such User Interface is compatible with the standards and requirements notified by the PAF Network from time to time, including by posting the same at [insert URL].
+
+9. The User Interface Provider shall ensure that, at all times, its User Interface:
+
+	i. prominently displays the following text:
+
+	“_Enjoy smoother navigation without annoying you with pop-ups. [PAF Network Name] personalizes your experience without directly identifying you. Opt-out or make changes at any time. By saving your preferences, this site’s privacy policy [insert URL to privacy policy] will apply.”_
+
+	_“Learn more about the data [PAF Network Name] uses and its accountability by
+visiting its Privacy Policy [hyperlinked to PAF Network Privacy Notice that is the privacy notice of PAF Operators at [insert URL]]_;
+
+	ii. includes, at a minimum, the following Preference-related options (including opt-in mechanism and text without amendment) and ensures that any preference option defaults to no/opted-out and permits Users to easily and immediately access and read the text marked with asterisks below, where ‘easily’ means the text is legible and
+located near what it describes:
+
+	*  a) [_A mechanism that a User can use to reset their Pseudonymous Identifiers_]*
+
+	* b) _Turn on personalized marketing [opt-in mechanism] See more relevant content and ads**_
+
+	* _c) Turn on standard marketing [opt-in-mechanism] See generic content and ads***_
+
+	* _d) Customize your experience [available for both Personalized and Standard marketing that enables more granular selection of relevant processing purposes at clause 4.4]_
+
+	* e) Ask for preferences on each site I visit****_
+
+	* _f) Apply your chosen preferences to other devices by inserting your email address and a code (applies only to devices on which you’ve taken the same steps) [mechanism and description that allows User to provide an email address and code, and allows this Pseudonymous Identifier to be reset without changing that email address]_
+	*  _g) Refuse all [mechanism that closes the User Interface without opting the User into any marketing – none of purposes in clause 4.4 shall apply if the User selects this option. If you decide to also include a ‘close dialog’ or similar option, this must have the same effect]_
+
+	* _(h) Save [mechanism that saves the Preferences selected by the User]_ 
+	
+	*[PAF Network Name]*  links your preferences to a Random ID. Your ID resets after 6 months. You can reset it sooner at any time by clicking the refresh icon._
```

Comment:
missing "_" at the start of the line
remove "*" just after [PAF Network Name] (the asterisk at the start of the line should actually be displayed).